receiver/windowseventlogreceiver/README.md

```diff
@@ -26,14 +26,15 @@ Tails and parses logs from windows event log API using the [opentelemetry-log-co
 | `attributes`                        | {}           | A map of `key: value` pairs to add to the entry's attributes.                                                                                                                                                                                  |
 | `resource`                          | {}           | A map of `key: value` pairs to add to the entry's resource.                                                                                                                                                                                    |
 | `operators`                         | []           | An array of [operators](https://github.com/open-telemetry/opentelemetry-log-collection/blob/main/docs/operators/README.md#what-operators-are-available). See below for more details                                                            |
-| `raw`                               | false        | If true, the windows events are not processed and sent as XML. If used in combination with `exclude_providers`, each event will be processed in order to determine its provider name.                                                          |
+| `raw` | false | If false, the body of emitted log records will contain a structured representation of the event. Otherwise, the body will be the original xml string. |
```

Comment:
```suggestion
| `raw` | false | If false, the body of emitted log records will contain a structured representation of the event. Otherwise, the body will be the original XML string. |
```